streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/controller/MenuController.java

```diff
@@ -53,8 +53,8 @@ public class MenuController {
     private CommonService commonService;
 
     @PostMapping("router")
-    public RestResponse getUserRouters() {
-        ArrayList<VueRouter<Menu>> routers = this.menuService.getUserRouters(commonService.getCurrentUser());
+    public RestResponse getUserRouters(@NotBlank(message = "{required}") Long teamId) {
+        ArrayList<VueRouter<Menu>> routers = this.menuService.getUserRouters(commonService.getCurrentUser().getUserId(), teamId);
```

Comment:
The code can be further improve:
`ArrayList<VueRouter<Menu>> routers = this.menuService.getUserRouters(commonService.getUserId(), teamId);`